pkg/tracking/sentry.go

```diff
@@ -67,6 +68,10 @@ func (hook *Hook) Fire(entry *logrus.Entry) error {
 
 	sentry.CaptureEvent(event)
 
+	if entry.Level <= logrus.FatalLevel {
```

Comment:
I believe we need to `Flush` only in case of the `Fatal`, thus 
```suggestion
	if entry.Level == logrus.FatalLevel {
```

Reply:
@Neurostep `Panic` level crashes the program as well, I believe we should include it.

Reply:
@laynax [Panic](https://github.com/sirupsen/logrus/blob/dd1b4c2e81afc5c255f216a722b012ed26be57df/entry.go#L387) does not crash or calls the `os.Exit`, only [Fatalf](https://github.com/sirupsen/logrus/blob/dd1b4c2e81afc5c255f216a722b012ed26be57df/entry.go#L381-L384) does. In this case, we should only check for equality here

Reply:
@laynax Actually I am wrong, `Panicf` calls [panic](https://github.com/sirupsen/logrus/blob/dd1b4c2e81afc5c255f216a722b012ed26be57df/entry.go#L260) under the hood, TIL. We are good here then 👍